packages/web/src/common/store/pages/track/sagas.ts

```diff
@@ -163,18 +163,10 @@ function* watchFetchTrack() {
             forceRetrieveFromSource
           })
         } else {
-          const ids =
-            canBeUnlisted && slug && handle
-              ? [{ id: trackId, url_title: slug, handle }]
-              : [trackId]
+          const ids = canBeUnlisted
```

Comment:
@dylanjeffers tagging you in this PR only for this part.

requiring slug & handle is not necessary for this to work, and probably hasn't been working as written for a while, but in the ts change we added the && slug && handle constraint, which aren't passed in via navigation, only when something is deep linked.

Separately, there was an additional retrieveTracks call for no reason

Reply:
Not sure I understand @raymondjacobson - if they're not necessary, why the ternary at all? Isn't the point of this line that we retrieve by only the track id if we don't have slug and title?

Reply:
The backend no longer needs them in order to serve back a hidden track. Isaac made that change a long time ago. They used to be required, but we made changes somewhere else that don't always provide them when canBeUnlisted = true

Reply:
So the typescripting of this file totally broke loading stems on mobile across the board

Reply:
I'll make a ticket to rip out this code though, it's very misleading

Reply:
That part makes sense, but I'm not sure what this code is accomplishing as it reads after your change - do we not want to retrieve by `[{id, url_title, handle}]` or `[id]` in all cases then?

Reply:
ah sorry github comments didn't load in